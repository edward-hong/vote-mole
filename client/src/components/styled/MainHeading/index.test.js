import React from 'react'
import { render } from '@testing-library/react'

import MainHeading from '.'

describe('MainHeading', () => {
	it('renders correctly', () => {
		const {
			container: { firstChild: element },
		} = render(<MainHeading>Main Heading</MainHeading>)

		expect(element).toMatchSnapshot()
	})
})
