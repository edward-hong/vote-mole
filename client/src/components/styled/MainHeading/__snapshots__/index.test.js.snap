// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
.c0 {
  text-align: center;
}

<h1
  class="c0"
>
  Main Heading
</h1>
`;
